test/quantization/test_quantize_fx.py

```diff
@@ -634,6 +634,7 @@ def forward(self, x):
         self.assertEqual(model_device, device)
 
     @skipIfNoFBGEMM
+    @unittest.skip("dict doesnot work in fx?")
```

Comment:
is it related to this PR?

Reply:
this test fails after the change, will need to ask FX team to help debug the issue